cmd/action/env.go

```diff
@@ -11,7 +11,11 @@ import (
 var envCmd = &cobra.Command{
 	Use:   "env",
 	Short: "Print atlas env params",
-	Long: `"ATLAS_NO_UPDATE_NOTIFIER": On any command, the CLI will check for updates with the GitHub public API once
+	Long: `Env prints atlas environment information.
+Every set environment param will print in the form of NAME=VALUE.
+
+List of supported environment parameters:
+"ATLAS_NO_UPDATE_NOTIFIER": On any command, the CLI will check for updates with the GitHub public API once
 	every 24 hours. To cancel this behavior, set the environment parameter "ATLAS_NO_UPDATE_NOTIFIER".`,
```

Comment:
Can we remove this indent ? It looks odd

Reply:
sure, fixed